app/views/pages/developer-guide/about-platformos/about-platformos.liquid

```diff
@@ -49,7 +49,7 @@ To help you quickly develop GraphQl queries, we provide a [GraphiQL](https://git
 {% include 'alert/tip', content: 'We did some user research with developers who were new to platformOS but have now built their first couple of projects. These are their tips that will get you up to speed in platformOS quicker and easier:
 
 * Focus on GraphQL before anything else, it is the key to platformOS. You will be able to craft complex queries in GraphQL and also send back only the data that you need. Later, discover fragment queries, a favorite feature of seasoned platformOS developers.
-* Once you are familiar with the basics of platformOS (development workflow, pages, layouts, assets), invest some time in understanding [Models](/developer-guide/models/models-model-schemas). You can relate Models to GraphQL and build custom endpoints.
+* Once you are familiar with the basics of platformOS (development workflow, pages, layouts, assets), invest some time in understanding [Records](/developer-guide/records/records-tables). You can relate Records to GraphQL and build custom endpoints.
```

Comment:
I imagine `/developer-guide/records/records` should be enough

Reply:
@slashek We try to follow the actual title of the topic as closely as possible, and here the topic title is Records and Tables. Should we change the topic title as well? I think users might search for tables, too. 
What do you think?

Reply:
As discussed, we will merge this PR now.